frontend/portals/ProductTaxDisclaimerAfter/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withPageProductId } from '@shopgate-ps/pwa-extension-kit/connectors';
```

Comment:
we should use the PWA HOC for this https://developer.shopgate.com/references/engage/hocs#withcurrentproduct

Reply:
done